internal/services/datafactory/data_factory_linked_service_azure_sql_database_resource.go

```diff
@@ -171,6 +171,27 @@ func resourceDataFactoryLinkedServiceAzureSQLDatabase() *pluginsdk.Resource {
 					Type: pluginsdk.TypeString,
 				},
 			},
+
+			"credential": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"reference_name": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+						},
+
+						"type": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+						},
+					},
+				},
+			},
```

Comment:
In `azurerm_data_factory_integration_runtime_managed` we just expose this as `credential_name` since `type` only has one valid value. To be consistent with other data factory resources can we please update this to 
```suggestion
			"credential_name": {
				Type:     pluginsdk.TypeString,
				Optional: true,
				ValidateFunc: validation.StringIsNotEmpty,
			},
```

Reply:
@stephybun thanks for the feedback 🙂  I've made the changes you requested.